src/main/java/de/dennisguse/opentracks/adapters/MarkerResourceCursorAdapter.java

```diff
@@ -96,7 +99,7 @@ public void bindView(View view, Context context, Cursor cursor) {
             }
         }
 
-        ListItemUtils.setListItem(activity, view, false, true, iconId, R.string.image_marker, name, null, null, 0, time, false, category, description, hasPhoto);
+        ListItemUtils.setListItem(activity, view, false, true, iconId, R.string.image_marker, name, null, null, 0, OffsetDateTime.ofInstant(Instant.ofEpochMilli(time), ZoneId.systemDefault()), category, description, hasPhoto);
```

Comment:
Can we extract this `OffsetDateTime.ofInstant(Instant.ofEpochMilli(time), ZoneId.systemDefault())`?
do we have DateUtils?

Reply:
We don't have any utility class for dates. Anyway, I think it's an atomic instruction, only a constructor of an OffsetDateTime, is it worth extract such a simple instruction to an utility class?

That line, now is: `OffsetDateTime.ofInstant(Instant.ofEpochMilli(time), track.getZoneOffset())`